Makefile

```diff
@@ -7,6 +7,9 @@ source:
 	# this is run from inside our collectd fork so we clone a clean copy to work off of
 	git clone ../../ collectd-$(VERSION).git
 	pushd collectd-$(VERSION).git ; \
+	if [[ $(DISTRO) == "epel-6" ]]; then \
```

Comment:
Just curious, why is this only needed in Cent OS 6?

Reply:
We should really be doing this on all platforms, but it's _required_ on CentOS 6 because of https://github.com/Stackdriver/collectd/pull/138.

Reply:
Actually, your intuition was right, and we do need this on other platforms as well. See #46.